write.js

```diff
@@ -1,20 +1,20 @@
-import Crc32 from './crc.js'
```

Comment:
browser requires to know the full path

Reply:
Ryan Dahl regretted a support for loading modules without extension
https://www.youtube.com/watch?v=M3BM9TB-8yA

and i agree with that now that browser start supporting import/export

Reply:
Yeah I agree that the extension is better

Reply:
@jimmywarting I set the rule to require extensions